client/me/help/help-contact/index.jsx

```diff
@@ -64,6 +65,10 @@ const HelpContact = React.createClass( {
 			this.props.connectHappychat();
 		}
 
+		if ( config.isEnabled( 'help/directly' ) ) {
+			this.props.initializeDirectly();
```

Comment:
From my understanding, this initialization performs the load of the directly scripts asynchronously. The [relevant line](https://github.com/Automattic/wp-calypso/pull/11489/files#diff-6b63894c947be4e7d604e6f6fbbeb9dbR83) is in #11489:

```js
loadScript( DIRECTLY_RTM_SCRIPT_URL, function( error ) {
	if ( error ) {
		reject( error );
	} else {
		resolve();
	}
} );
```